versions/draft-3/SPEC.md

```diff
@@ -274,6 +277,12 @@ For more details on the postfix quantifiers, see the section on [Optional Parame
 
 For more information on type and how they are used to construct commands and define outputs of tasks, see the [Data Types & Serialization](#data-types--serialization) section.
 
+#### Explanation of FileURI
+
+The FileURI represents a File that could be either a local file or a file in an object store.  
```

Comment:
I'd say `... represents a path to a file which could be either local or in a remote object store`

Reply:
Done